lib/src/main/java/grpcbridge/parser/Serializer.java

```diff
@@ -14,5 +15,5 @@
 
     Function<RpcMessage, HttpResponse> serializeAsync(JsonFormat.Printer printer);
 
-    boolean supportsAny(Collection<String> accepted);
+    boolean supportsAny(Collection<MediaType> accepted);
```

Comment:
Right now these are all just "hard coded" to utf-8 because this is the default in java and also the recommend charset on the web in general. This could be refined more in the future by inspecting the `Accept-Charset` header, but that might be overkill for now